src/utils/object.js

```diff
@@ -17,11 +17,18 @@ export const parseKeyValues = (object = {}) =>
       })
 
 // ['key: value'] -> {key: 'value'}
-export const generateKeyValues = (chips = []) => {
+export const generateKeyValues = (data = []) => {
   const keyValuePairs = {}
 
-  chips.forEach(chip => {
-    const [key, value] = chip.split(': ')
+  data.forEach(dataItem => {
+    const key = dataItem.replace(/:.*$/g, '')
+    let value = dataItem.replace(/.*: /g, '')
+
+    if (dataItem.includes(': {')) {
+      value = dataItem.replace(/.*: {/g, '{')
+    } else if (dataItem.includes(': [')) {
+      value = dataItem.replace(/.*: \[/g, '[')
+    }
 
     keyValuePairs[key] = value
```

Comment:
I deleted my last comment, but there's still a little something we can do here.
Instead of `if-else if` we can use ternary operator to get the value, then instead of `let` we can use `const`. And actually we can even remove the `value` variable completely.

```js
    keyValuePairs[key] =
      dataItem.includes(': {') ? dataItem.replace(/.*: {/g, '{')  :
      dataItem.includes(': [') ? dataItem.replace(/.*: \[/g, '[') :
      /* else */                 dataItem.replace(/.*: /g, '')
```
I like this formatting for nested ternaries, because it's like a table with two columns where on the left you have a condition and on the right you have the corresponding value in case this condition passes.
But since Prettier enforces this formatting:
```js
    keyValuePairs[key] = dataItem.includes(': {')
      ? dataItem.replace(/.*: {/g, '{')
      : dataItem.includes(': [')
      ? dataItem.replace(/.*: \[/g, '[')
      : dataItem.replace(/.*: /g, '')
```
which I like much less, maybe we should just stick to the `if-else if`. 😕  I'm not sure.